aml/org.testeditor.aml.dsl/src/org/testeditor/aml/dsl/Aml.xtext

```diff
@@ -53,12 +53,12 @@ ComponentType returns ComponentType:
 ;
 
 ComponentElement returns ComponentElement:
-	'element' name=ID 'is' type=[ComponentElementType|QualifiedName] ('{'
+	'element' name=ID 'is' type=[ComponentElementType|QualifiedName] (('{'
 		('label' '=' label=STRING)?
 		('locator' '=' locator=STRING)?
 		('locatorStrategy' '=' locatorStrategy=[types::JvmEnumerationLiteral|QualifiedName])?
 		(valueSpaceAssignments+=ValueSpaceAssignment)*
-	'}')?
+	'}')|('locate' 'by' locatorStrategy=[types::JvmEnumerationLiteral|QualifiedName] ('using')? locator=STRING)(valueSpaceAssignments+=ValueSpaceAssignment)*)?
```

Comment:
I'm not sure about the `valueSpaceAssignments`. Do we really want them in this short syntax?
If yes, a parser test is missing for them.